Authentication-System/authen/app/models.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+class CustomUser(AbstractUser):
+
+    contact_number = models.CharField(max_length=15, blank=True)
```

Comment:
Contact number should have aa unique check.

Reply:
yes added